README.md

```diff
@@ -1,90 +1,114 @@
 
-### Current ROCm Version: 2.4
+### Current ROCm Version: 2.5
 
-- [New features and enhancements in ROCm 2.4](#new-features-and-enhancements-in-rocm-24)
-- [The latest ROCm platform - ROCm 2.4](#the-latest-rocm-platform---rocm-240)
+- [New features and enhancements in ROCm 2.5](#new-features-and-enhancements-in-rocm-25)
+- [The latest ROCm platform - ROCm 2.5](#the-latest-rocm-platform-rocm-25)
 - [Hardware Support](#hardware-support)
   * [Supported GPUs](#supported-gpus)
   * [Supported CPUs](#supported-cpus)
   * [Not supported or limited support under ROCm](#not-supported-or-limited-support-under-rocm)
-- [Supported Operating Systems](#supported-operating-systems---new-operating-systems-available)
+- [Supported Operating Systems](#supported-operating-systems-new-operating-systems-available)
   * [ROCm support in upstream Linux kernels](#rocm-support-in-upstream-linux-kernels)
 - [Installing from AMD ROCm repositories](#installing-from-amd-rocm-repositories)
   * [ROCm Binary Package Structure](#rocm-binary-package-structure)
-  * [Ubuntu Support - installing from a Debian repository](#ubuntu-support---installing-from-a-debian-repository)
+  * [Ubuntu Support - installing from a Debian repository](#ubuntu-support-installing-from-a-debian-repository)
   * [CentOS/RHEL 7 (7.4, 7.5, 7.6) Support](#centosrhel-7-74-75-76-support)
-- [Known issues / workarounds](#known-issues--workarounds)
+- [Known issues / workarounds](#known-issues-workarounds)
 - [Closed source components](#closed-source-components)
 - [Getting ROCm source code](#getting-rocm-source-code)
   * [Installing repo](#installing-repo)
   * [Downloading the ROCm source code](#downloading-the-rocm-source-code)
   * [Building the ROCm source code](#building-the-rocm-source-code)
-- [Deprecation Notice](#deprecation-notice)
+- [Deprecation Notice](#deprecation-notice-hcc)
 - [Final notes](#final-notes)
 
-### New features and enhancements in ROCm 2.4
+### New features and enhancements in ROCm 2.5
 
-#### TensorFlow 2.0 support
-ROCm 2.4 includes the enhanced compilation toolchain and a set of bug fixes to support TensorFlow 2.0 features natively
+#### UCX 1.6 support
+Support for UCX version 1.6 has been added.
 
-#### AMD Infinity Fabric&#x2122; Link enablement 
-ROCm 2.4 adds support to connect two Radeon Instinct MI60 or Radeon Instinct MI50 boards via AMD Infinity Fabric&#x2122; Link GPU interconnect technology.
+#### BFloat16 GEMM in rocBLAS/Tensile
+Software support for BFloat16 on Radeon Instinct MI50, MI60 has been added.  This includes:
+- Mixed precision GEMM with BFloat16 input and output matrices, and all arithmetic in IEEE32 bit
+- Input matrix values are converted from BFloat16 to IEEE32 bit, all arithmetic and accumulation is IEEE32 bit. Output values are rounded from IEEE32 bit to BFloat16
+- Accuracy should be correct to 0.5 ULP
+
+#### ROCm-SMI enhancements
+Support for querying the memory size, driver version, and firmware version has been added to ROCm-smi
```

Comment:
CLI support for querying the memory size, driver version, and firmware version and LIB support for memory size and driver version have been added to ROCm-smi.